src/edu/washington/escience/myria/api/encoding/CreateFunctionEncoding.java

```diff
@@ -5,14 +5,17 @@
 
 import java.util.Set;
 
-import edu.washington.escience.myria.Schema;
+import edu.washington.escience.myria.MyriaConstants.FunctionLanguage;
 
 /**
  *
  */
 public class CreateFunctionEncoding extends MyriaApiEncoding {
   @Required public String name;
-  @Required public String definition;
-  @Required public Schema outputSchema;
+  @Required public String description;
+  @Required public String outputSchema;
```

Comment:
because it is sent in from python as a string and not  a type enumeration, makes the code simpler..